server/plugin/oAuth.go

```diff
@@ -215,6 +229,23 @@ func (p *Plugin) StoreOAuthToken(mattermostUserID string, oauthTokenFormValues u
 	return nil
 }
 
+// isAccessTokenExpired checks if a user's access token is expired
+func (p *Plugin) isAccessTokenExpired(mattermostUserID string) bool {
```

Comment:
```suggestion
// IsAccessTokenExpired checks if a user's access token is expired
func (p *Plugin) IsAccessTokenExpired(mattermostUserID string) bool {
```

Reply:
Shouldn't we make them capital from starting, as we need to change them later while making test cases so that they are not private while mocking them?